tachyon/math/polynomials/univariate/univariate_sparse_polynomial_unittest.cc

```diff
@@ -40,11 +40,7 @@ TEST_F(UnivariateSparsePolynomialTest, IsZero) {
   EXPECT_TRUE(Poly().IsZero());
   EXPECT_TRUE(Poly::Zero().IsZero());
 
-  Poly p = Poly(Coeffs({{0, GF7(0)}, {1, GF7(0)}}));
-  p.coefficients().RemoveZeros();
-  EXPECT_TRUE(p.IsZero());
```

Comment:
With this PR, we are enforcing polynomial construction with only _cleaned up_ coefficients. So this test fails in line 43 because the constructor checks if the coefficients are clean.

Reply:
I reverted the removal and fixed the test.